zdm/scripts/repeaters.py

```diff
@@ -0,0 +1,191 @@
+""" 
+This script creates zdm grids and plots localised FRBs
+
+It can also generate a summed histogram from all CRAFT data
+
+"""
+import os
+from pkg_resources import resource_filename
+from zdm import cosmology as cos
+from zdm import misc_functions
+from zdm import parameters
+from zdm import survey
+from zdm import pcosmic
+from zdm import iteration as it
+from zdm.craco import loading
+from zdm import io
+from zdm import repeat_grid as rep
+
+import numpy as np
+from zdm import survey
+from matplotlib import pyplot as plt
+
+def main():
```

Comment:
I don't think this script does much of anything.
Remove this file?

Reply:
You are correct - I've now updated it to be useful